D6StarWars/D6StarWars.html

```diff
@@ -162,12 +161,12 @@
             </select>
           </label>
         </div>
-        <div class='col'>
+        <div class="sheet-col">
           <table class="custom-roll-table">
             <tr>
-              <th style="vertical-align: top;">Initiative bonus</th>
-              <td colspan="2" style="padding-left: 6px; padding-bottom: 40px">
-                <input class="smallnumber shortnumber" min="0" name="attr_initiative" type="number" value="0"/>D+<input class="pipnumber shortnumber" max="2" min="0" name="attr_initiativepip" type="number" value="0"/>
+              <th class="init">Initiative bonus</th>
+              <td colspan="2" class="init">
+                <input class="init" min="0" name="attr_initiative" type="number" value="0"/>D+<input class="init" max="2" min="0" name="attr_initiativepip" type="number" value="0"/>
```

Comment:
Every time the sheet is initialized, this includes via pop-out, there is a chance that all sheet values will be reset to default value. What wonderful coding!

Reply:
Sorry to say but this should not be true and I have never encountered this behaviour. Could you please send a video/gif that shows this resetting behaviour? I would really appreciate that.

EDIT:
The original sheet (v2.32 form February) has all the defaults (with _3D+0_).

EDIT:
In every frontend framework I worked with (including Roll20) the value attribute defines only a default value that WILL be overwritten any incoming external data (e.g.: database or manual input).

Reply:
Speak of the fucking devil. The values absolutely **ARE** fucked and we've got another person present to show your buggerup. I don't even need to waste the time making a video of a new sheet. You are wrong. You have fucked up. You did in fact, screw up. Quit trying to save face and fucking revert the PR and submit a public apology already.

Reply:
I'll also note you haven't disputed you've fucked the blocks sizings.